### ISSUE TITLE
A11y : Cannot access variables warnings via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46495]
**Steps to reproduce**:

From [notes](https://docs.google.com/spreadsheets/d/1e3sjrB3wjgQfttK5ANNkac5wmDwLszhgW0fO1kWT2LI/edit#gid=0):

> Not used variable warning is not accessible

**Actual Result**:

Not all warnings are accessible / focusable via keyboard navigation

**Expected Result**

workflow accessible for keyboard-only users

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by natellium** [comment]
Needs some UX exploration and discussion.
